scripts/performance-tests-kontrol.sh

```diff
@@ -10,6 +10,7 @@ KONTROL_VERSION=${KONTROL_VERSION:-'v0.1.49'}
 SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
```

Comment:
Also while we are here, `KONTROL_VERSION` is still hard-wired to 0.1.49 in this script and should be switched to `master` as in the `kevm` script (also changing how we build things).